book/scheduling.md

```diff
@@ -510,9 +510,6 @@ function __runXHROnload(body, handle) {
 }
 ```
 
-As you can see, tasks allow not only the browser but also applications
-running in the browser to delay tasks until later.
-
```

Comment:
Ah, the reason for this paragraph is that, I think everywhere, we never end a section with a code block. Not actually sure that this rule matters but that's what sometimes creates these sort-of useless end section paragraphs. Often I fix these by pulling forward the first paragraph of the next section, though that doesn't always work.